frontend/src/components/processMeasurements/ProcessMeasurementsDetailContent.js

```diff
@@ -46,6 +52,15 @@ const ProcessMeasurementsDetailContent = ({
       listPropertyValues({ object_id__in: processMeasurement.id, content_type__model: "processmeasurement" });
     }
 
+    const processIsLoaded = processMeasurement?.process in processesByID;
+    const process = processesByID[processMeasurement?.process] || {};
+    if (!processIsLoaded) {
+      listProcesses({id__in: processMeasurement?.process});
```

Comment:
i would use a "get"  here since there is a single one...

Reply:
is there overhead associated with fetching a single item with `list`?